docs/node-configuration.md

```diff
@@ -247,6 +247,7 @@ protocol-related settings described in the table below.
 | MaxBlockSystemFee | `int64` | `900000000000` | Maximum overall transactions system fee per block. |
 | MaxTraceableBlocks | `uint32` | `2102400` | Length of the chain accessible to smart contracts. | `RemoveUntraceableBlocks` should be enabled to use this setting. |
 | MaxTransactionsPerBlock | `uint16` | `512` | Maximum number of transactions per block. |
+| MaxValidUntilBlockIncrement | `uint32` | `5760` | Upper increment size of blockchain height in blocks exceeding that a transaction should fail validation. It is set to estimated daily number of blocks with 15s interval by default. |
```

Comment:
"Upper height increment limit for transaction's ValidUntilBlock field value relative to the current blockchain height, exceeding which a transaction will fail validation"... @EliChin would love something like that.

Reply:
Fixed.